src/test/java/org/example/Operations/DataObjectOperationsTest.java

```diff
@@ -512,188 +545,172 @@ public void test_modifying_metadata_atomically() {
+            assertEquals("tempZone", permissions.path("zone").asText());
+            assertEquals("rodsuser", permissions.path("type").asText());
+            assertEquals("read_object", permissions.path("perm").asText());
+
+            // Remove the data object.
+            DataObjectRemoveParams params = new DataObjectRemoveParams();
+            params.setNoTrash(1);
+            params.setAdmin(1);
+            res = rods.dataObject().remove(rodsToken, dataObject, 0, params);
+            assertEquals("remove data object request failed", 200, res.getHttpStatusCode());
+            assertEquals("remove data object failed", 0,
+                    getIrodsResponseStatusCode(res.getBody()));
+        } finally {
+            // Reset permissions
+            List<ModifyPermissionsOperations> jsonParam = new ArrayList<>();
+            jsonParam.add(new ModifyPermissionsOperations("alice", "null"));
+            rods.dataObject().modifyPermissions(rodsToken, dataObject, jsonParam, OptionalInt.empty());
+
+            // Remove the data object.
+            DataObjectRemoveParams params = new DataObjectRemoveParams();
+            params.setNoTrash(1);
+            params.setAdmin(1);
+            rods.dataObject().remove(rodsToken, dataObject, 0, params);
         }
-
-        // Remove the data object.
-        DataObjectRemoveParams params = new DataObjectRemoveParams();
-        params.setNoTrash(1);
-        params.setAdmin(1);
-        res = rods.dataObject().remove(token, data_object, 0, params);
-        assertEquals("remove data object request failed", 200, res.getHttpStatusCode());
-        assertEquals("remove data object failed", 0,
-                getIrodsResponseStatusCode(res.getBody()));
     }
 
     @Test
-    public void test_modifying_replica_properties() {
+    public void testModifyingReplicaProperties() {
+        Response res;
         String directoryName = "/tempZone/home/rods";
         String fileName = "modrepl.txt";
-        String data_object = directoryName + "/" + fileName;
-
-        // Create a data object.
-        res = rods.dataObject().touch(token, data_object, null);
-        assertEquals("touch request failed", 200, res.getHttpStatusCode());
-        assertEquals("touch failed", 0,
-                getIrodsResponseStatusCode(res.getBody()));
-
-        // Show the replica is currently marked as good and has a size of 0.
-        String query = String.format(
-                "select DATA_REPL_STATUS, DATA_SIZE where COLL_NAME = '%s' and DATA_NAME = '%s'",
-                directoryName, fileName
-        );
-        res = rods.queryOperations().executeGenQuery(token, query, null);
-        assertEquals("execute_genquery request failed", 200, res.getHttpStatusCode());
-        assertEquals("execute_genquery failed", 0,
-                getIrodsResponseStatusCode(res.getBody()));
+        String dataObject = directoryName + "/" + fileName;
```

Comment:
`fileName` should be `dataObjectName`.

Reply:
Sorry, do you mean as in, renaming 'fileName' to 'dataObjectName'?

Reply:
That's exactly what I was referring to.

Please mark this as resolved.